src/fides/api/schemas/user.py

```diff
@@ -46,7 +46,7 @@ def validate_password(cls, password: str) -> str:
             raise ValueError("Password must have at least one capital letter.")
         if re.search("[a-z]", decoded_password) is None:
             raise ValueError("Password must have at least one lowercase letter.")
-        if re.search(r"[\W]", decoded_password) is None:
+        if re.search(r"[\W_]", decoded_password) is None:
```

Comment:
Great, can you add another test similar to `test_create_user`?
https://github.com/ethyca/fides/blob/main/tests/ops/api/v1/endpoints/test_user_endpoints.py#L145

That test has a request body of `body = {"username": "test_user", "password": str_to_b64_str("TestP@ssword9")}`

Your test can use a password instead that has an underscore -

Reply:
Thanks for the suggestion, just added it!